From c6319da05828f7e18aa7599088b893dc84d74cd4 Mon Sep 17 00:00:00 2001
From: Seb Laveze <sebastien.laveze@nxp.com>
Date: Wed, 23 Aug 2017 10:55:00 +0200
Subject: [PATCH 2543/5242] MLK-16473 irqchip: imx-irqsteer: change irqsteer
 type to level

commit  f64d93fbce5268ac39f7db401ffb07ee242ce05c from
https://source.codeaurora.org/external/imx/linux-imx.git

Irqsteer block only supports level interrupts.

(BuildInfo: SCFW 3e70523d, IMX-MKIMAGE 0, ATF 0)

Signed-off-by: Seb Laveze <sebastien.laveze@nxp.com>
Signed-off-by: Fugang Duan <fugang.duan@nxp.com>
Reviewed-by: Richard Zhu <hongxing.zhu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/irqchip/irq-imx-irqsteer.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/irqchip/irq-imx-irqsteer.c b/drivers/irqchip/irq-imx-irqsteer.c
index 27afb23..2e8f04a 100644
--- a/drivers/irqchip/irq-imx-irqsteer.c
+++ b/drivers/irqchip/irq-imx-irqsteer.c
@@ -84,7 +84,7 @@ static int imx_irqsteer_irq_map(struct irq_domain *h, unsigned int irq,
 				irq_hw_number_t hwirq)
 {
 	irq_set_chip_data(irq, h->host_data);
-	irq_set_chip_and_handler(irq, &imx_irqsteer_irq_chip, handle_edge_irq);
+	irq_set_chip_and_handler(irq, &imx_irqsteer_irq_chip, handle_level_irq);
 
 	return 0;
 }
-- 
1.7.9.5

